spec/amp-cors-requests.md

```diff
@@ -368,6 +368,59 @@ Our response headers would be:
 Access-Control-Allow-Credentials: true
 Access-Control-Allow-Origin: https://example-com.cdn.ampproject.org
 ```
+## Working with cached fonts
+Google AMP Cache caches AMP HTML documents, images and fonts to optimize the speed of the AMP page. While making the AMP page fast, we also want to be careful in securing the cached resources. We will be making a change in how AMP cache responds it’s cached resources, 
+typically for fonts, by respecting the origin’s `Access-Control-Allow-Origin` value.
+
+### How AMP Cache works as-is
+When an AMP page was loading [https://example.com/some/font.ttf](https://example.com/some/font.ttf) from `@font-face src` attribute, AMP Cache will cache the font file and serve the resource as below with having the wild card `Access-Control-Allow-Origin`.
+
+- URL [https://example-com.cdn.ampproject.org/r/s/example.com/some/font.tff](https://example-com.cdn.ampproject.org/r/s/example.com/some/font.tff)
+- Access-Control-Allow-Origin: *
+
+### Font implementation 
+While the current implementation is permissive, this could lead to unexpected use of the fonts from cross-origin sites. In this change AMP Cache will start to respond with the exact same `Access-Control-Allow-Origin` value the origin server responds. 
+To properly load the fonts from the cached AMP document, you will need to accept the AMP Cache origin via the header.
+
+A sample implementation would be:
+
+```javascript
+function assertFontCors(req, res, opt_validMethods, opt_exposeHeaders) {
+  var unauthorized = 'Unauthorized Request';
+  var allowedOrigins = [
+     "https://example.com",
+     "https://example-com.cdn.ampproject.org"];
+
+  // If allowed CORS origin
+  if (allowedOrigins.indexOf(req.headers.origin) != -1) {
+      res.setHeader('Access-Control-Allow-Origin', req.headers.origin);
+  } else {
+      res.statusCode = 401;
+      res.end(JSON.stringify({message: unauthorized}));
+      throw unauthorized;
+  }
+}
+```
+As an example, if you wanted to load /some/font.ttf in https://example.com/amp.html, the origin server should respond with the Access-Control-Allow-Origin header as below.
+
+<amp-img alt="CORS font example" layout="responsive" src="https://amp.dev/static/img/docs/cors-font.jpg" width="2268" height="1594">
```

Comment:
Does this image exist?

Reply:
It will with the next amp.dev deploy
https://github.com/ampproject/amp.dev/pull/3174